src/ansys/pyensight/core/utils/dsg_server.py

```diff
@@ -55,6 +68,63 @@ def reset(self, cmd: Any = None) -> None:
         if cmd is not None:
             self.hash.update(cmd.hash.encode("utf-8"))
         self.cmd = cmd
+        self._material = None
+
+    def _parse_material(self) -> None:

```

Comment:
Here I parse the JSON from the protocol buffer to get more material details.  We do support the notion that the JSON can contain multiple representations for different scheme (e.g. MDL vs EnSight vs ...)